packages/@okta/vuepress-site/docs/reference/api/oie-policies/index.md

```diff
@@ -0,0 +1,737 @@
+| id       | String  | Unique identifier for this Policy (read-only).                                               |
+| name     | String  | Human-readable name for the Policy, configurable during creation or updating.                |
+| type     | String  | Type of the policy. For IdP Routing Policy objects, this needs to be `Okta:IdentifierMatch`. |
+| status   | String  | `ACTIVE`  or  `INACTIVE`.                                                                    |
+| default  | Boolean | `true` for the first instance of this policy, which gets created by default.                 |
+
+### Identifier Match Policy Object Example
+
+```json
+{
+    "id": "rst10y1rmKOjickDM0g4",
+    "name": "Default Policy",
+    "type": "Okta:IdentifierMatch",
+    "status": "ACTIVE",
+    "default": true,
+    "_links": {
+        "self": {
+            "href": "https://idx.okta1.com/api/v1/policies/rst10y1rmKOjickDM0g4",
+            "hints": {
+                "allow": [
+                    "GET",
+                    "PUT"
+                ]
+            }
+        },
+        "rules": {
+            "href": "https://idx.okta1.com/api/v1/policies/rst10y1rmKOjickDM0g4/rules",
+            "hints": {
+                "allow": [
+                    "GET"
+                ]
+            }
+        }
+    }
+}
+```
+## Identifier Match Rule Object
+
+| Property    | Type                                                                                  | Description                                                                                                        |
+|-------------|---------------------------------------------------------------------------------------|--------------------------------------------------------------------------------------------------------------------|
+| name        | String                                                                                | Human-readable name for the Rule, configurable during creation or updating.                                        |
+| id          | String                                                                                | Unique identifier for this Rule (read-only).                                                                       |
+| type        | String                                                                                | Type of the policy. For Identifier Match Rule Objects, this needs to be `Okta:IdentifierMatch`.                    |
+| priority    | Integer                                                                               | Used to determine which Rules take precedence.                                                                     |
+| conditions  | Array                                                                                 | Identifier and App conditions are supported for this Rule type.                                                    |
```

Comment:
curious if we shall/can link to Identifier conditions and App conditions?

Reply:
I suggest we include examples in the json request below, here's an example with both supported conditions for this policy rule:
```
    "conditions": [
        {
            "key": "Okta:Identifier",
            "op": "STRING_ENDS_WITH",
            "value": "idx.com"
        },
        {
            "key": "Okta:AppInstance",
            "op": "IN_LIST",
            "value": [
            	"{{appInstanceId}}"
        	]
        }
    ],
```
would you mind include those into the example below @motikagan-okta ?

Reply:
I think people would certainly wonder when are supported values for `op` for Identifier conditions and App condition respectively. I'd prefer to be more specific (maybe a separated section to list details of all conditions ?)

Reply:
Change description of `conditions` property to:
Identifier and App conditions are supported for this Rule type.
Added conditions to sample.

Reply:
Added a mention that `conditions` objects are documented on existing Policies API page and a link to the section:

The `conditions` objects, which can be specified in Rule objects, are documented in the existing Policy API documentation at: <https://developer.okta.com/docs/reference/api/policy/#conditions>